index.njk

```diff
@@ -10,17 +10,6 @@ eleventyNavigation:
     <div class="cta-gradient button"><a href="/info/become-a-member">Become a Member</a></div>
 </section>
 
-<section class="container" id="special-content">
-    <h1>Ask Me Anything (AMA) 11 April 2022</h1>
-    
-    <div><img src="/img/2022-04-ama.png" alt="Save the Date: Rust's Q2 AMA will take place online, Monday April 11 2022 at 10:30am EDT / 7:30am PDT / 4:30pm CET" /></div>
-    
-    <h3>Join Rust Foundation Executive Director Bec Rumbul and colleagues in an Ask-Me-Anything (AMA) interactive webinar focusing on our <a href="/grants/">Community Grants Program</a>. Moderated by Sage Griffin.</h3>
-    
-    <div class="cta-gradient button"><a href="https://us06web.zoom.us/j/86938717294">Here's the link to join.</a></div>
-    
-</section>
-
 <section class="container" id="special-content">
```

Comment:
Even though you removed the `special-content div img` CSS above, I guess the resizing here works because of the global `img` CSS change you made. I think 🤔 

Anyway, I guess we keep the `special-content` id here because of its use for font, etc? I wonder what happens if we removed the id here. Would things break?